doc/source/python_teaser_ray.py

```diff
@@ -0,0 +1,18 @@
+import time
+import ray
+
+ray.init(num_cpus=4)
+
+@ray.remote
+def f():
+    time.sleep(1)
+    return True
+
+start = time.time()
+
+futures = [f.remote() for i in range(8)]
```

Comment:
Side note: do you guys want to have a consistent message on "object IDs" versus "futures"?

Reply:
We like to call them "Object IDs" mostly and tell people who know about futures that they are basically futures. The term "Object ID" is better because "future" typically doesn't mean that you would treat it as data, make it part of a datastructure or anything like that.

Reply:
Agreed that "object IDs" is more technically correct.

Reply:
sure